### PR TITLE
Update direct-routing-translate-numbers.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5665
Fixed wording in PowerShell.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-translate-numbers.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The policy is applied at the SBC level. You can assign multiple translation rule
...
+To assign, configure, and list number manipulation rules on SBCs, use the [New-CSOnlinePSTNGateway](https://docs.microsoft.com/powershell/module/skype/new-csonlinepstngateway) and [Set-CSOnlinePSTNGateway](https://docs.microsoft.com/powershell/module/skype/set-csonlinepstngateway) cmdlets together with the  InboundTeamsNumberTranslationRules, InboundPSTNNumberTranslationRules, OutboundTeamsNumberTranslationRules, OutboundPSTNNumberTranslationRules, InboundTeamsNumberTranslationRules, InboundPSTNNumberTranslationRules, OutboundTeamsNumberTranslationRules, and OutboundPSTNNumberTranslationRules parameters.
 
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-translate-numbers.md:45]:
```suggestion
> [!NOTE]
```

File: Teams/direct-routing-translate-numbers.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ The policy is applied at the SBC level. You can assign multiple translation rule
...
 > The maximum total number of translation rules is 400, maximum translation parameter name length is 100 symbols, maximum translation parameter pattern length is 1024 symbols and maximum translation parameter translation length is 256 symbols.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-translate-numbers.md:46]:
```suggestion
> The maximum total number of translation rules is 400, maximum translation parameter name length is 100 symbols, maximum translation parameter pattern length is 1024 symbols, and maximum translation parameter translation length is 256 symbols.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - not related to this PR, but please also include the two minor additional edits.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**